libraries/matchers.rb

```diff
@@ -24,7 +24,7 @@
     oneview_drive_enclosure:            %i[hard_reset patch refresh set_uid_light set_power_state],
     oneview_enclosure:                  %i[add remove refresh reconfigure patch],
     oneview_enclosure_group:            standard_actions + [:set_script],
-    oneview_ethernet_network:           standard_actions + [:reset_connection_template],
+    oneview_ethernet_network:           standard_actions + %i[reset_connection_template add_scope remove_scope replace_scopes patch],
```

Comment:
Since the 3 actions will be recurring and default, create a new var like the `standard_actions`